### PR TITLE
Tg connection

### PR DESCRIPTION
**Pull request by domenicoblanco**
The bot can now connect to Telegram, the only required step is to set an environment variable called `QDBotToken`

### CODE REVIEW

**Review by Valeri0p — COMMENTED** [review]
Looks good to me at a first glance 👍

**Comment by domenicoblanco** [comment]
Completed both points in #1

File: main.py [hunk 1]
```diff
@@ -1,2 +1,12 @@
-from telegram import Update
-from telegram.ext import ApplicationBuilder
\ No newline at end of file
+from os import getenv
+from telegram.ext import ApplicationBuilder, MessageHandler, filters
+
+def main() -> None:
+    token = getenv("QDBotToken")
+    app = ApplicationBuilder().token(token).build()
+    app.add_handler(MessageHandler(filters.ChatType.GROUPS & (filters.StatusUpdate.NEW_CHAT_MEMBERS), None))
```
Comment (Valeri0p) [inline main.py:7]: `(filters.StatusUpdate.NEW_CHAT_MEMBERS)
Are those those brackets really necessary?`

**Review by Valeri0p — APPROVED** [review]
Ready to merge.

**Pull request merged**